src/Sulu/Component/Cache/CacheInterface.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Cache;
+
+/**
+ * Interface for cache.
```

Comment:
A little bit more description would be nice (maybe you can mention that this cache doesn't rely on other files)